go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/ory/herodot v0.9.13
 	github.com/ory/jsonschema/v3 v3.0.7
 	github.com/ory/keto/proto v0.10.0-alpha.0
-	github.com/ory/x v0.0.502
+	github.com/ory/x v0.0.524-0.20221229213023-29c9e1da3e4d
```

Comment:
Is the needed functionality not simply in `0.0.524`?

Reply:
yep, changed.